omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -14,22 +14,17 @@
     <%  if (showLastViewedPatients && !doInitialSearch) {
             lastViewedPatients.each { it -> %>
 
-    /* [RA-452]
-    Data pushed to lastViewedPatients must be sanitized with both ui.escapeJs and ui.encodeHtmlContent.
-    - escapeJS because untrusted data is being passed into js code
-    - encodeHtmlContent because data is eventually being displayed on the page
-     */
-    lastViewedPatients.push({
-        uuid:"${ ui.escapeJs(ui.encodeHtmlContent(it.uuid)) }",
-        name:"${ it.personName ? ui.escapeJs(ui.encodeHtmlContent(ui.format(it.personName))) : '' }",
-        gender:"${ ui.escapeJs(ui.encodeHtmlContent(it.gender)) }",
-        // it.age is of type int (doesn't need sanitization)
-        age:"${ it.age ?: '' }",
-        birthdate:"${ it.birthdate ? ui.escapeJs(ui.encodeHtmlContent(dateFormatter.format(it.birthdate))) : '' }",
-        // it.birthdateEstimated is of type boolean (doesn't need sanitization)
-        birthdateEstimated: ${ it.birthdateEstimated },
-        identifier:"${ it.patientIdentifier ? ui.escapeJs(ui.encodeHtmlContent(it.patientIdentifier.identifier)) : '' }"
-    });
+                lastViewedPatients.push({
+                    uuid:"${ ui.encodeHtml(it.uuid) }",
+                    name:"${ it.personName ? ui.encodeHtml(ui.format(it.personName)) : '' }",
+                    gender:"${ ui.encodeHtml(it.gender) }",
```

Comment:
Yeah, that's intentional. I figured a single call to encodeHtml is better than two calls to escapeJS and then encodeHtmlContent.